### PR TITLE
Run scripts

### PR DESCRIPTION
**Pull request by suganya-s**
Kind of a WIP but would be good to get feedback

updated/added some logging,`run_all.py` orchestrates all the scripts (WIP as waiting for the building ingestion module also), Makefile has `make test` now

### CODE REVIEW

File: src/run_all.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+if __name__ == '__main__':
+    logging.config.fileConfig('src/logging.conf')
+    module_logger = logging.getLogger('run_all')
+
+    functional_edges_and_weights = multiprocessing.Process(
+        name='make_functional_edges_and_weights', target=make_functional_edges_and_weights)
+    content_store_data = multiprocessing.Process(name='get_content_store_data', target=get_content_store_data)
+
```
Comment (ellieking17) [inline src/run_all.py]: Nice!

**Comment by ellieking17** [comment]
> Kind of a WIP but would be good to get feedback
> 
> updated/added some logging,`run_all.py` orchestrates all the scripts (WIP as waiting for the building ingestion module also), Makefile has `make test` now

**Comment by ellieking17** [comment]
This is looking really good @suganya-s.

**Comment by suganya-s** [comment]
Waiting on https://github.com/alphagov/govuk-related-links-recommender/pull/17 and https://github.com/alphagov/govuk-related-links-recommender/pull/18, then this should run ok in AWS.

**Comment by suganya-s** [comment]
@karlbaker02 could you take a look at this, and then we can try running `python3 src/run_all.py` to run all the scripts

File: Makefile [hunk 2]
```diff
@@ -25,66 +20,50 @@ requirements: test_environment
...
-endif
-		@echo ">>> New conda env created. Activate with:\nsource activate $(PROJECT_NAME)"
-else
-	$(PYTHON_INTERPRETER) -m pip install -q virtualenv virtualenvwrapper
-	@echo ">>> Installing virtualenvwrapper if not already intalled.\nMake sure the following lines are in shell startup file\n\
-	export WORKON_HOME=$$HOME/.virtualenvs\nexport PROJECT_HOME=$$HOME/Devel\nsource /usr/local/bin/virtualenvwrapper.sh\n"
-	@bash -c "source `which virtualenvwrapper.sh`;mkvirtualenv $(PROJECT_NAME) --python=$(PYTHON_INTERPRETER)"
-	@echo ">>> New virtualenv created. Activate with:\nworkon $(PROJECT_NAME)"
-endif
+# ## Upload Data to S3
```
Comment (karlbaker02) [inline Makefile]: Should we delete all of these commented-out commands or are they still useful?

Comment (suganya-s) [reply Makefile]: I was keeping them in in case they helped with your copying stuff to/from S3 at all, are they useful or shall I delete?

**Pull request merged**